glass-easel/src/behavior.ts

```diff
@@ -1275,14 +1275,20 @@ export class Behavior<
         this._$propertyMap[name] = d
         initValueFuncs.push({
           name,
-          func: d.default === undefined ? () => simpleDeepCopy(d.value) : d.default,
+          func: () => {
+            if (typeof d.default === 'function') {
```

Comment:
This check should better be checked outside of the function (I guess)?

Reply:
fixed